sklearn/model_selection/tests/test_split.py

```diff
@@ -1619,3 +1620,79 @@ def test_random_state_shuffle_false(Klass):
     with pytest.raises(ValueError,
                        match='has no effect since shuffle is False'):
         Klass(3, shuffle=False, random_state=0)
+
+
+def test_group_time_series_fail_groups_are_none():
+    """
+    The GroupTimeSeriesSplit with no group should raise an Error
+    """
+    X = [[1, 2], [3, 4], [5, 6], [7, 8], [9, 10], [11, 12], [13, 14]]
+
+    # Should fail if the 'groups' is None
+    assert_raises_regexp(ValueError,
```

Comment:
This is using the `unittest` testing framework, which I think we no longer use. New tests should be implemented using the current new test framework: `pytest`. Maybe some other person can confirm this.

Just look at the test above this one, to see how to implement it.

Reply:
Done